sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/projects/models/_models_py3.py

```diff
@@ -0,0 +1,1769 @@
+        code: Union[str, "ErrorCode"],
+        message: str,
+        target: Optional[str] = None,
+        details: Optional[List["Error"]] = None,
+        innererror: Optional["InnerErrorModel"] = None,
+        **kwargs
+    ):
+        """
+        :keyword code: Required. One of a server-defined set of error codes. Possible values include:
+         "InvalidRequest", "InvalidArgument", "Unauthorized", "Forbidden", "NotFound",
+         "ProjectNotFound", "OperationNotFound", "AzureCognitiveSearchNotFound",
+         "AzureCognitiveSearchIndexNotFound", "TooManyRequests", "AzureCognitiveSearchThrottling",
+         "AzureCognitiveSearchIndexLimitReached", "InternalServerError", "ServiceUnavailable".
+        :paramtype code: str or ~azure.ai.language.questionanswering.projects.models.ErrorCode
+        :keyword message: Required. A human-readable representation of the error.
+        :paramtype message: str
+        :keyword target: The target of the error.
+        :paramtype target: str
+        :keyword details: An array of details about specific errors that led to this reported error.
+        :paramtype details: list[~azure.ai.language.questionanswering.projects.models.Error]
+        :keyword innererror: An object containing more specific information than the current object
+         about the error.
+        :paramtype innererror: ~azure.ai.language.questionanswering.projects.models.InnerErrorModel
+        """
+        super(Error, self).__init__(**kwargs)
+        self.code = code
+        self.message = message
+        self.target = target
+        self.details = details
+        self.innererror = innererror
+
+
+class ErrorResponse(msrest.serialization.Model):
+    """Error response.
+
+    :ivar error: The error object.
+    :vartype error: ~azure.ai.language.questionanswering.projects.models.Error
+    """
+
+    _attribute_map = {
+        'error': {'key': 'error', 'type': 'Error'},
+    }
+
+    def __init__(
+        self,
+        *,
+        error: Optional["Error"] = None,
+        **kwargs
+    ):
+        """
+        :keyword error: The error object.
+        :paramtype error: ~azure.ai.language.questionanswering.projects.models.Error
+        """
+        super(ErrorResponse, self).__init__(**kwargs)
+        self.error = error
+
+
+class Errors(msrest.serialization.Model):
+    """Collection of Error types.
+
+    :ivar errors:
+    :vartype errors: list[~azure.ai.language.questionanswering.projects.models.Error]
+    """
+
+    _attribute_map = {
+        'errors': {'key': 'errors', 'type': '[Error]'},
+    }
+
+    def __init__(
+        self,
+        *,
+        errors: Optional[List["Error"]] = None,
+        **kwargs
+    ):
+        """
+        :keyword errors:
+        :paramtype errors: list[~azure.ai.language.questionanswering.projects.models.Error]
+        """
+        super(Errors, self).__init__(**kwargs)
+        self.errors = errors
+
+
+class ExportJobResultUrl(msrest.serialization.Model):
```

Comment:
It looks like this should be the result of the `export` LRO poller?

Reply:
yes!